app/scripts/modules/core/delivery/executionGroup/execution/execution.directive.js

```diff
@@ -81,7 +81,11 @@ module.exports = angular
         header: 'Really delete execution?',
         buttonText: 'Delete',
         body: '<p>This will permanently delete the execution history.</p>',
-        submitMethod: () => executionService.deleteExecution(this.application, this.execution.id)
+        submitMethod: () => executionService.deleteExecution(this.application, this.execution.id).then(()=>{
```

Comment:
add space between `()`, `=>`, and `{` to make it consistent with everything else, i.e. `() => {`